k8s/jiva/snapshot-cleanup.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+usage()
+{
+    echo "Usage: bash script.sh <controller_pod_name> <application_namespace> <numbber_of_snapshots_to_delete>"
+    exit 1
+}
+
+delete_jiva_snapshot()
+{
+ctrl_pod_name=$1
+application_namespace=$2
+number_of_snapshot=$3
+count=0
+snapshot_list_cmd='kubectl exec -it $ctrl_pod_name -n $application_namespace -- jivactl snapshot ls | grep -v ID | wc -l'
+snapshot_name_cmd='kubectl exec -it $ctrl_pod_name -n $application_namespace -- jivactl snapshot ls | grep -v ID | awk 'NR==2''
+
+if [ $(eval $snapshot_list_cmd) -le $((number_of_snapshot)) ]; then
```

Comment:
This should be <= (number_of_snapshot - 2) since first row contains "ID", second row contains latest snapshot and we won't allow deletion of latest snapshot.

Reply:
@utkarsh2102 Snapshot list command contain `grep -v ID`. So the list will not have ID. The condition should be <= (number_of_snapshot - 1)